### ISSUE TITLE
Start block at 5%

### ISSUE THREAD

**Issue by DRAIXxXx** [0xSpoofed/Vingod#4]
hi i have see the last issue and i have the same problem it's possible to new patch please ?

**Comment by 0xSpoofed** [comment]
Yes an update will come, when I don't know, right now I don't really have the time. 
Have a nice day.
